tests/unit_test/queue/test_zmq.py

```diff
@@ -106,8 +106,8 @@ async def benchmark_queue(qps, ip=None, port=None):
         signal.alarm(0)
 
 @pytest.mark.asyncio
-async def test_queue_zmq(setup_ray_env):
+@pytest.mark.parametrize("qps", [128.0, 256.0, 512.0, 1024.0])
+async def test_queue_zmq(setup_ray_env, qps):
```

Comment:
How about setting the QPS to a single maximum value solely for testing correctness?

Reply:
I think zmq is a very critical component of Llumnix which should be tested multiple times.

Reply:
but we just test the correctness here. I think the maximum value is enough.